pom.xml

```diff
@@ -49,15 +49,15 @@
     </licenses>
 
     <scm>
-        <connection>scm:git:git://github.com/jgroups-extras/native-s3-ping.git</connection>
-        <developerConnection>scm:git:ssh://git@github.com/jgroups-extras/native-s3-ping.git</developerConnection>
+        <connection>scm:git:git://github.com/jgroups-extras/jgroups-aws.git</connection>
```

Comment:
Does this mean that the repo will be renamed after this is merged ?

Reply:
@tristantarrant That's the intention as per the proposal in the email.

Reply:
Great